README.md

```diff
@@ -99,7 +99,7 @@ If you are working offline or do not have access to a cluster with the CRDs inst
 you may supply a directory to use as a search path for types:
 
 ```sh
-kubectl validate ./my_crd.yaml --local-schemas ./path/to/folder/with/crds
+kubectl validate ./my_crd.yaml --local-crds ./path/to/folder/with/crds
```

Comment:
Think we also need to add a section to document `local-schemas`

Reply:
Will do

Reply:
Done